### DISCUSSION TITLE
Quick Tags support for old editor?

### DISCUSSION THREAD

**Discussion by asshuka** [AprilSylph/XKit-Rewritten#382] [Suggestion Box]
As of now I still prefer the old editor, but Quick Tags is a must for mass reblogging from outside the dashboard (I have blogs saved in RSS feeds and there's too many to move to Tumblr's subscription feed, it's a whole thing). I'm not sure if this hasn't yet been implemented or featured in the tracking board because of restrictions on Tumblr's end, so I'm posting here instead of the Issues tab.

**Comment by AprilSylph** [comment]
XKit Rewritten exclusively targets Tumblr's new web stack, which the legacy post editor is not a part of. Continue using XKit 7 to affect the legacy post editor.
